@xen-orchestra/web-core/lib/components/DonutChart.vue

```diff
@@ -0,0 +1,102 @@
+<!-- v1.1 -->
+<template>
+  <div class="wrapper">
```

Comment:
Is there a reason to add a wrapper?

Also, the class name of the root element should be named after the component name.